patterns/services-team-photos.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Title: Services - Team photos
+ * Slug: twentytwentyfive/services-team-photos
+ * Categories: banner, call-to-action, featured
+ * Description: Team photos in services section.
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:group {"align":"wide","layout":{"type":"default"}} -->
+<div class="wp-block-group alignwide"><!-- wp:columns -->
+<div class="wp-block-columns"><!-- wp:column -->
+<div class="wp-block-column"><!-- wp:heading {"level":3} -->
+<h3 class="wp-block-heading"><?php echo esc_html_x( 'Our small team is a group of driven, detail-oriented people who are passionate about their customers.', 'Team Photos', 'twentytwentyfive' ); ?></h3>
+<!-- /wp:heading --></div>
+<!-- /wp:column -->
+
+<!-- wp:column -->
+<div class="wp-block-column"><!-- wp:columns -->
+<div class="wp-block-columns"><!-- wp:column -->
+<div class="wp-block-column"><!-- wp:image {"aspectRatio":"1","scale":"cover","sizeSlug":"full","linkDestination":"none"} -->
+<figure class="wp-block-image size-full"><img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/team_photo_1.jpg" alt="Team Member One" style="aspect-ratio:1;object-fit:cover"/></figure>
```

Comment:
@devmuhib009 Could you please use WebP images instead of JPGs? 
Additionally, could you include an images license in the `readme.txt` file as recommended in the [above comment](https://github.com/WordPress/twentytwentyfive/pull/126#issuecomment-2315216937)?